e2e/test/scenarios/question/reproductions/42065-mongo-filter-by-id.cy.spec.js

```diff
@@ -0,0 +1,58 @@
+import { WRITABLE_DB_ID } from "e2e/support/cypress_data";
+import { restore, openTable } from "e2e/support/helpers";
+
+describe(
+  "issue 42010 -- Unable to filter by mongo id",
+  { tags: "@mongo" },
+  () => {
+    beforeEach(() => {
+      restore("mongo-5");
+      cy.signInAsAdmin();
+
+      cy.request(`/api/database/${WRITABLE_DB_ID}/schema/`).then(({ body }) => {
+        const tableId = body.find(table => table.name === "orders").id;
+        openTable({
+          database: WRITABLE_DB_ID,
+          table: tableId,
+          mode: "notebook",
+        });
+      });
+    });
+
+    it("should be possible to filter by Mongo _id column (metabase#42010)", () => {
+      cy.get("button").contains("Visualize").click();
```

Comment:
If you omit the `mode: "notebook"` from the `openTable`, you can remove this step.